tests/test_release_transpiler.py

```diff
@@ -42,3 +46,21 @@ def test_transpile(tmpdir):
                 assert os.path.isfile(
                     "{}-{}-{}.yml".format(release_base, py_ver, rhel_ver)
                 )
+
+
+def test_transpile_for_pip(tmpdir):
+    release_file = os.path.join(_get_test_root(), "data", "test_release_matrix.yml")
+    repo_file = os.path.join(_get_test_root(), "data", "test_repository.yml")
+    release_base = os.path.basename(release_file).strip(".yml")
+    not_pip_pkg = "lib3"
+    expected_line = "lib2==2.3.4"
+    with tmpdir.as_cwd():
+        transpile_releases_for_pip(release_file, os.getcwd(), repo_file)
+        for rhel_ver in ("rhel7",):
+            for py_ver in ("py38",):
```

Comment:
This is an interesting way of writing:
```python
rhel_ver = "rhel7"
py_ver = "py38"
```

Reply:
indeed! i think it comes from a time when there were several potential versions of OS and python. if i scrap it here, i'd probably have to scrap it many places in the tests and/or code... what do you think? should i keep it consistent, do lots of changes, ... ?

Reply:
It is only a matter of time before we add py310 ...

Reply:
great! and i guess we will support both 3.8 and 3.10, at least for a while?